_data/users.yml

```diff
@@ -417,6 +417,20 @@
   links:
     - urltext: View the Hudson River Trading website
       url: https://www.hudsonrivertrading.com/
+- name: HugoBank
+  highlight: false
+  anchor: hugobank
+  logo: /assets/images/logos/hugobank.png
```

Comment:
I guessed if this was an accurate anchor

Reply:
yep